lib/watcher_adapter.js

```diff
@@ -5,12 +5,25 @@ const sane = require('sane');
 const logger = require('heimdalljs-logger')('broccoli:watcherAdapter');
 const SourceNode = require('./wrappers/source-node');
 
+const eventMap = {
+  add: 'added',
+  delete: 'deleted',
+  change: 'changed',
+};
+
 function defaultFilterFunction(name) {
   return /^[^.]/.test(name);
 }
 
 function bindFileEvent(adapter, watcher, node, event) {
   watcher.on(event, (filepath, root) => {
+    // This is added to keep compatiblity for ember-cli
+    // as it relied on broccoli-sane-watcher to log it:
+    // https://github.com/ember-cli/broccoli-sane-watcher/blob/48860/index.js#L158
+    if (adapter.options.verbose) {
+      console.log(`file ${eventMap[event]} ${filepath}`);
```

Comment:
What happened to https://github.com/ember-cli/ember-cli/commit/23793804213dad14d833556f71ebe212458c59fc#diff-9ff56008b40607a31bc04df7f41906cbR113 ?

Reply:
This won't go via the `ui` package in ember-cli, I don't think we should do this here but do what was in that commit above that seems to have been removed

Reply:
https://github.com/ember-cli/ember-cli/pull/8679